README.md

```diff
@@ -1,48 +1,5 @@
-# curriculum-databases-projects-template
+# FCC Test Suite Template
```

Comment:
Use Template to complete README

Reply:
- [ ] OPTIONAL: This comment looks a bit commanding than suggesting. So, it is better to write it in a more polite and suggestive word. Also, you did not specify which template you are suggesting and it would be great if you give the link to the template in your comment. :pray: